### ISSUE TITLE
Structure of the compact navigation explanation

### ISSUE THREAD

**Issue by jonaslil** [w3c/publ-a11y#482]
I see a structural problem in the compact navigation explanation, [Example 12](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/guidelines/#example-compact-explanation]).

"Supports page navigation" works well as a standalone phrase - which is how I think the different navigation aids should be handled (see https://github.com/w3c/publ-a11y/issues/460#issuecomment-2488620497). However, it does not fit into the concatenated explanation.

The current display guide logic could potentially produce "Navigation by supports page navigation," which highlights the issue. The other substrings ("table of contents," "headings," "index") are all noun phrases that work well in the context. If the concatenation approach is maintained, the word "supports" must be removed from the page navigation string, along with "navigation," which is already mentioned at the start. A more suitable phrase would be "Navigation by ... page numbers".

**Comment by mattgarrish** [comment]
> "Supports page navigation" works well as a standalone phrase

Is it understandable to end users? It lacks the context of the descriptive phrase: "A page list enables users to navigate directly to pages from the identified print source version."

Even that seems a bit misleading, as I don't see that the techniques check whether it's a print source or not. This could be displayed if there are auto-generated static page breaks and the pageBreakSource property is set to "none".

In any case, any ebook supports page navigation if you only understand page navigation as what the reading system provides for its virtual pagination. A little more context, like "supports navigation to persistent page breaks", might help here.

**Comment by jonaslil** [comment]
I fully agree that "page navigation" isn’t understandable to non-specialists without explanatory context.

"Supports navigation to persistent page breaks" is on the right track, but I’d like to note two things after discussing this with my colleagues:

- Persistent page breaks exist mostly at the code level in reflowable EPUBs, which makes me hesitant to use the term here.
- *Page numbers* are what users interact with in the page navigation feature and what’s needed to compare a reflowable EPUB to a static source. To explain page navigation to end users, we need to include the word "page numbers."

I would propose something like: Navigation using static/permanent/persistent page numbers.

I suggested a similar phrasing in an earlier, now closed issue, but the editors decided on the current version of the short explanation.

**Comment by GeorgeKerscher** [comment]
I think this issue is resolved by putting the information in a list. I think the use of go to page also addresses this problem.
